src/Guesser/FilterTypeGuesser.php

```diff
@@ -60,7 +60,7 @@ public function guessType($class, $property, ModelManagerInterface $modelManager
             switch ($mapping['type']) {
                 case ClassMetadata::ONE:
                 case ClassMetadata::MANY:
-                    $options['operator_type'] = EqualType::class;
+                    $options['operator_type'] = EqualOperatorType::class;
```

Comment:
this change is correct?

Reply:
Yes, see https://github.com/sonata-project/SonataDoctrineORMAdminBundle/blob/3.x/src/Guesser/FilterTypeGuesser.php#L62